app/views/layouts/application.html.erb

```diff
@@ -13,5 +13,7 @@
 
   <body>
     <%= yield %>
+    <p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>
```

Comment:
A quoi elles servent ces lignes ?

Reply:
c'est un message d'alerte pour prévenir si tu as signed-in ou pas